DonjinKrawler-Client/src/main/java/donjinkrawler/adapter/WavPlayer.java

```diff
@@ -0,0 +1,22 @@
+package donjinkrawler.adapter;
+
+import javax.sound.sampled.AudioSystem;
+import javax.sound.sampled.Clip;
+import java.io.File;
+
+public class WavPlayer implements AdvancedMediaPlayer {
+    @Override
+    public void playMp3(String fileName) {}
+
+    @Override
+    public void playWav(String fileName) {
+        try {
+            File audioFile = new File(ClassLoader.getSystemResource("sounds/".concat(fileName)).getFile());
```

Comment:
Wouldn't this load the sound on each call to playWav? That'd may delay the sounds badly.